geo/src/algorithm/map_coords.rs

```diff
@@ -41,28 +41,28 @@ mod modern {
         ///
         /// ```
         /// use geo::MapCoords;
-        /// use geo::Point;
+        /// use geo::{Coordinate, Point};
         /// use approx::assert_relative_eq;
         ///
         /// let p1 = Point::new(10., 20.);
-        /// let p2 = p1.map_coords(|(x, y)| (x + 1000., y * 2.));
+        /// let p2 = p1.map_coords(|Coordinate { x, y }| Coordinate { x: x + 1000., y: y * 2. });
         ///
         /// assert_relative_eq!(p2, Point::new(1010., 40.), epsilon = 1e-6);
         /// ```
         ///
         /// You can convert the coordinate type this way as well
```

Comment:
Unrelated, but we should probs advertise `Convert` here now?

Reply:
I've rebased, and pushed up a new commit which references `Convert` for the case of simply changing numeric type , and then updated this example to solve a more complex problem involving both type conversion and a transformation for which `Convert` would be insufficient.